app/scripts/stores/mainStore.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import { observable, computed, action, map } from 'mobx';
+import { observable, computed, action, map, extendObservable } from 'mobx';
```

Comment:
Are you ever using extendObservable? If not, remove this.

Reply:
Mostly Notes for myself
Push up fixes for this